File: Data Structures & Algorithms/remove-element/submission-0.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int ans = 0;
        for(int i=0; i<nums.size(); i++) {
            if(nums[i]==val) continue;
            nums[ans]=nums[i];
            ans++;
        }
        return ans;
    }
};

File: Data Structures & Algorithms/jump-game/submission-0.py
class Solution:
    def canJump(self, nums: List[int]) -> bool:
        if len(nums) == 1:
            return True
            
        canReach = set()
        canReach.add(len(nums)-1)
        for i in range(len(nums)-2,-1,-1):
            x = i+nums[i]
            if x in canReach:
                if i == 0:
                    return True
                canReach.add(i)

        return False